library/registry

```diff
@@ -2,7 +2,7 @@ Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
-Tags: 2.8.0, 2.8, 2, latest
+Tags: 2.8.1, 2.8.0, 2.8, 2, latest
```

Comment:
2.8.0 should not be in here

Reply:
Right

Reply:
https://github.com/docker-library/official-images/pull/12001/commits/3989b41fa61384db20809af75b9e9d1e46e56ba8